### PR TITLE
Add mali to marriage abroad

### PR DESCRIPTION
**Pull request by CodeSonia**
Added Mali to list of countries and made corresponding changes on outcomes, view the [Trello card here](https://trello.com/c/aZ9fShYR/344-add-mali-to-getting-married-abroad-tool)

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

**Comment by CodeSonia** [comment]
Content designer is happy with the changes, however awaiting approval from embassy before merging 👍 hence "Do not merge"

**Review by Rosa-Fox — APPROVED** [review]
👍

**Pull request merged**
